app/controllers/api/v1/people_actions.rb

```diff
@@ -0,0 +1,89 @@
+module Api::V1
+  module PeopleActions
+    PERSON_ATTRIBUTES = [
+      :first_names,
+      :last_name,
+      :date_of_birth,
+      :gender_additional_information,
+      assessment_answers: [%i[key
+                              date
+                              expiry_data
+                              category
+                              title
+                              comments
+                              assessment_question_id
+                              nomis_alert_type
+                              nomis_alert_type_description
+                              nomis_alert_code
+                              nomis_alert_description
+                              created_at
+                              expires_at
+                              imported_from_nomis]],
+      identifiers: [%i[value identifier_type]],
+    ].freeze
+    PERMITTED_PERSON_PARAMS = [:type, attributes: PERSON_ATTRIBUTES, relationships: {}].freeze
+    PERMITTED_FILTER_PARAMS = %i[police_national_computer prison_number].freeze
+
+    def index_and_render
+      prison_number = filter_params[:prison_number]
```

Comment:
this does not support `prison_number` comma-separated (see V2 implementation)
should we fix it?

Reply:
Not in this PR. Maybe never as is v1

Reply:
👍